### DISCUSSION TITLE
Save the file into a s3 bucket

### DISCUSSION THREAD

**Discussion by sdespont** [PHPOffice/PhpSpreadsheet#3457] [Q&A]
Hello,

I have to save generated Excel files to a S3 bucket instead of local storage. I am considering using the storage abstraction library https://flysystem.thephpleague.com

Is there any possibility to manage the way to save a file or the only solution is to save the file locally ?

Best regards

**Comment by oleibman** [comment]
PR #2326, responding to issue #2249, was added just over a year ago to permit saving to S3. I don't have any way to verify it, but the PR's creator and others confirmed that it worked correctly.

**Reply by sdespont** [reply]
OK thanks, I will have a look to this PR.
